features/facebookImport/rollup.config.js

```diff
@@ -71,7 +71,7 @@ export default (commandLineArgs) => {
                     JSON.stringify(
                         process.env.POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION ||
                             fallbackAuthorization
-                    ),
+                    ) || "",
```

Comment:
OK, this might make sense, but still, changes in `upload_all` (which are for uploading stuff to the app stores) are not really needed, and actually not run routinely.